src/brasil/gov/agenda/tests/test_subscribers.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+from brasil.gov.agenda.testing import INTEGRATION_TESTING
+from plone import api
+from zope.event import notify
+from zope.lifecycleevent import ObjectModifiedEvent
+
+import datetime
+import unittest
+
+
+class AgendaSubscriberTestCase(unittest.TestCase):
```

Comment:
Please review the name of this class; you are not testing a subscriber here, just if a behavior is enabled.

Reply:
and that's what the subscriber does: enable a behavior.